packages/schedule/src/ScheduleContainer.spec.js

```diff
@@ -15,89 +16,114 @@ import ScheduleContainer from './ScheduleContainer';
 jest.useFakeTimers();
 
 describe('ScheduleContainer', () => {
-  let wrapper;
   let requestAnimationFrame;
   let cancelAnimationFrame;
 
   const mockRaf = createMockRaf();
 
-  const basicExample = props => (
-    <ScheduleContainer {...props}>{elapsed => <p>{elapsed}</p>}</ScheduleContainer>
+  const BasicExample = props => (
+    <ScheduleContainer {...props}>
+      {elapsed => <p data-test-id="schedule">{elapsed}</p>}
+    </ScheduleContainer>
   );
 
   beforeEach(() => {
     requestAnimationFrame = jest
       .spyOn(window, 'requestAnimationFrame')
       .mockImplementation(mockRaf.raf);
     cancelAnimationFrame = jest.spyOn(window, 'cancelAnimationFrame');
-    setTimeout.mockClear();
+    clearTimeout.mockClear();
   });
 
   afterEach(() => {
     window.requestAnimationFrame.mockRestore();
     jest.clearAllTimers();
+    jest.clearAllMocks();
   });
 
   it('sets up requestAnimationFrame', () => {
-    wrapper = mount(basicExample());
+    render(<BasicExample />);
+
+    act(() => {
+      jest.runOnlyPendingTimers();
+    });
 
-    jest.runOnlyPendingTimers();
     expect(requestAnimationFrame).toHaveBeenCalled();
   });
 
   it('updates elapsed render prop on each raf call', () => {
-    wrapper = mount(basicExample());
+    const { getByTestId } = render(<BasicExample />);
+    const originalError = console.error;
+
+    console.error = jest.fn();
 
-    jest.runOnlyPendingTimers();
     act(() => {
+      jest.runOnlyPendingTimers();
+      // This throws a state error but I don't think it's an actual problem
+      // with the code itself since it has a clean up function
       mockRaf.step({ count: 60 });
```

Comment:
This throws an error like it has unmounted the component and is trying to update state? So I suppress the console.error call.

Reply:
This is fixed in bfe1708